app/domain/authentication/authn_jwt/authenticator.rb

```diff
@@ -0,0 +1,50 @@
+require 'command_class'
+
+module Authentication
+  module AuthnJwt
+    # Generic JWT authenticator that receive JWT vendor configuration and uses to validate that the authentication
+    # request is valid, and return conjur authn token accordingly
+    Authenticate = CommandClass.new(
+      dependencies: {
+        token_factory: TokenFactory.new
+      },
+      inputs: %i[jwt_configuration authenticator_input]
+    ) do
+
+      def call
+        validate_and_decode_token
+        get_conjur_id
+        validate_restrictions
+        new_token
```

Comment:
i asked this previously but the comment was removed. Where will the regular validations go? e.g validate the user can access the authenticator webservice?

Reply:
WIll be added out of scope of the dummy authenticator

Reply:
But they should be added here in def call